client/components/jetpack/card/jetpack-product-card/display-price/time-frame.tsx

```diff
@@ -18,10 +18,16 @@ const TimeFrame: React.FC< TimeFrameProps > = ( { expiryDate, billingTerm } ) =>
 
 	const billingTermText = useMemo( () => {
 		if ( billingTerm === TERM_MONTHLY ) {
-			return translate( '/month, billed monthly' );
+			return {
+				normal: translate( '/month, billed monthly' ),
+				compact: translate( '/mo, billed monthly' ),
```

Comment:
Can we add the comment as mentioned here https://github.com/Automattic/wp-calypso/pull/67634#discussion_r968209230?

Reply:
Thanks